### ISSUE TITLE
[PHP] Fails to load syntax file, nothing is highlighted

### ISSUE THREAD

**Issue by ccampbell** [sublimehq/Packages#187]
As a result of updating to the latest code here I am getting this at startup

![](https://devi.vimeocdn.com/craig/qI5q)

```
error: Error loading syntax file "Packages/PHP/PHP.sublime-syntax": Error in regex: invalid backref number/name in regex (?=^\2\b)
```

Interestingly it broke with commit 409ccb469bca8bcc2a5c70c562d1db3428d3015f

If I checkout commit a91eb66ef181399398950fe6470825b735905b85 it works fine. Not entirely sure how changing the javascript syntax broke php, but maybe it is something to do with the fact that a php file can have embedded javascript since it is essentially just html?

**Comment by wbond** [comment]
This is a bug that will be fixed with the next build of Sublime Text.

**Comment by wbond** [comment]
This should be resolved by c6cd372e5944e5078a03d1d47f6de146dbd8133d
